### ISSUE TITLE
systemd runners no longer working

### ISSUE THREAD

**Issue by ddiss** [rapido-linux/rapido#87]
The lrbd and mpath-local runners haven't been working for me since upgrading to Leap 15.1 . It seems that the "rd.systemd.unit=emergency" hook doesn't have any affect. I need to take a closer look at this one.

**Comment by ddiss** [comment]
Looks like systemd's kernel module load service may be responsible for the failed boot:
```
:/# journalctl -u systemd-modules-load.service
-- Logs begin at Thu 2019-08-22 15:01:58 UTC, end at Thu 2019-08-22 15:02:16 UTC. --
Aug 22 15:01:58 rapido1 systemd-modules-load[112]: Failed to find module 'scsi_dh_alua'
Aug 22 15:01:58 rapido1 systemd-modules-load[112]: Failed to find module 'scsi_dh_emc'
Aug 22 15:01:58 rapido1 systemd-modules-load[112]: Failed to find module 'scsi_dh_rdac'
Aug 22 15:01:58 rapido1 systemd-modules-load[112]: Failed to find module 'dm-multipath'
Aug 22 15:01:58 rapido1 systemd-modules-load[112]: Module 'sg' is builtin
Aug 22 15:02:15 rapido1 systemd[1]: systemd-modules-load.service: Main process exited, code=exited, status=1/FAILURE
Aug 22 15:02:15 rapido1 systemd[1]: Failed to start Load Kernel Modules.
Aug 22 15:02:15 rapido1 systemd[1]: systemd-modules-load.service: Unit entered failed state.
Aug 22 15:02:15 rapido1 systemd[1]: systemd-modules-load.service: Failed with result 'exit-code'.
```

These (unneeded) modules are pulled in via:
```
/usr/lib/modules-load.d/sg.conf
/usr/lib/modules-load.d/multipath.conf
```

I'm not sure which dracut component is responsible for placing these files.

**Comment by ddiss** [comment]
This should now be fixed in master via 3fef5c9139fdf654b64e21bf3af9870b4b651c7d and parent.
